QAChat/QA_Bot/qa_bot.py

```diff
@@ -52,8 +72,26 @@ def __answer_question_with_context(self, question: str, context: list[str]) -> s
             'The sky is blue during a clear day.'
         """
 
-        # For the moment simply return the context without the use of an LLM
-        return "\n".join(context)
+        context_str = "\n\n".join(f"### INFORMATION: {x}" for i, x in enumerate(context))
+
+        template = "### Instruction: You are a chatbot helping other people to answer questions." \
```

Comment:
This is then something we can fine-tune to get better results?

Reply:
yes